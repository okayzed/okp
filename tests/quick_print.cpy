#include <iostream>
#include <string>
int main()
	int a, b;
	std::string str;
	? "Type two integers: " a b "Well done! Now type a string: " str "Congratulations!\n"
	
	!
	! "Following line is produced by command: ! a b str \"Second string\""
	! a b str "Second string"
	!
	
	!! "!! is a quick print without ending the line..."
	! " Ending the line in the following command exemplified!"
	
	! "Following line is produced by command: ?? a b str"
	?? a b str
	
	//Cout still works, and can optionally imply "<<"
	std::cout "A is valued " a std::endl
	std::cout << "B is valued " << b << std::endl