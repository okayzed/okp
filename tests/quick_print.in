10 11 somestring
