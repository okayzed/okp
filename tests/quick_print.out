Type two integers: Well done! Now type a string: Congratulations!

Following line is produced by command: ! a b str "Second << ' ' << string"
10 11 somestring Second string

!! is a quick print without ending the line... Ending the line in the following command exemplified!
Following line is produced by command: ?? a b str
A is valued  10 

B is valued 11
